ui/eureka/eureka.js

```diff
@@ -8,6 +8,34 @@ let Options = {
   // 20 minutes for Ovni?
   SuppressPopMs: 60 * 20 * 1000,
   FlagTimeoutMs: 90000,
+  ZoneName: {
+    en: {
+      'Eureka Anemos': 'Eureka Anemos',
+      'Eureka Pagos': 'Eureka Pagos',
+      'Eureka Pyros': 'Eureka Pyros',
+      'Eureka Hydatos': 'Eureka Hydatos',
+    },
+    cn: {
+      'Eureka Anemos': '常风之地',
```

Comment:
Yes, zone names are in Chinese too.

Reply:
That's quite unfortunate.  None of the other languages have translated zone names, so I haven't had to handle that anywhere else.  I wonder if there'd be a way to get the English zone name and have the ffxiv plugin / cactbot send that so that cactbot doesn't have to handle any of this.  Maybe in the future.

Reply:
maybe you can read MapID from act plugin or memory?

Reply:
Yeah, that's something to consider for the future too.  I think the ffxiv plugin has it accessible.